### DISCUSSION TITLE
What about subscriptions?

### DISCUSSION THREAD

**Discussion by dodanex** [Diizzayy/nuxt-graphql-client#168] [Q&A]
Hi @Diizzayy ,
Does this module offer support for subscriptions? 
If yes, where is the documentation with code example for this?
If no, do you plan to support this in the near future?
Thanks

**Comment by raducretu** [comment]
I would like to use aws subscriptions

**Comment by dodanex** [comment]
@Diizzayy any news regarding this?

**Comment by Diizzayy** [comment]
@dodanex @raducretu Apologies for the late reply here, subscription support is around the corner.

Let me know if you're interested in testing the early release version.

**Reply by stlbucket** [reply]
Is this still something you plan to add?

**Reply by stlbucket** [reply]
https://github.com/Diizzayy/nuxt-graphql-client/pull/239

there are a few open issues on this repo about subscriptions - this one looks like it could be pending merge, but it's super old.
